openwisp_monitoring/monitoring/base/models.py

```diff
@@ -119,7 +119,7 @@ def content_type_key(self):
         except AttributeError:
             return None
 
-    def check_threshold(self, value, time=None):
+    def check_threshold(self, value, time=None, notify=True):
```

Comment:
Looks like it should be a local variable instead of a parameter

Reply:
I thought upon it that this could be helpful in tests, similar to the `store` parameter in `perform_check`. Wdyt, @PabloCastellano, can it be helpful in testing?

Reply:
is this argument being used at all anywhere? It looks like is not. If that's the case, it should be removed. We must refrain from adding code we don't really need.